binary.com.conf

```diff
@@ -5,6 +5,8 @@ server {
     add_header Cache-Control "public, max-age=7200, s-maxage=600, must-revalidate";
     charset UTF-8;
 
+    root /usr/share/nginx/html;
+    index index.html index.htm;
```

Comment:
is this assuming inside en, there is index.html?, or redirecting it to en/home?

Reply:
this is not inside `en` instead this is inside `dist` which the root directory for static contents.